Program.cs

```diff
@@ -75,68 +71,90 @@ public static void UpdatePerson()
                         case 1:
                             Console.WriteLine("Please Enter The New Name!");
                             var newName = Console.ReadLine();
-                            person.setFirstName(newName);
+                            people[id].SetFirstName(newName);
```

Comment:
`people[id]` is too dangerous!!!
First, you should check the id exists or not.
Then you can use the code below for finding the user.
`var people.FirstOrDefault(p => p.Id == id);`